server/src/com/gip/xyna/xprc/xfractwfe/python/PythonMdmGeneration.java

```diff
@@ -419,7 +421,7 @@ private XynaObjectInformation loadXynaObjectInfo(Long revision, String fqn, bool
         result.methods = loadOperations(((DOM) doe).getOperations());
       }
     } catch (Exception e) {
-      throw new RuntimeException(e);
+      return null;
```

Comment:
It need alot ressources to build Stacktraces. We should avoid to throw Exceptions in regular cases. Is there any way to find out, that the datatype is not deployed bevore the Exception is thrown?

Reply:
I don't think this will be a common path. Xmom-state information is not tracked by the xmomdatabase afaik, so we can't modify the select. We could check the file or consult the DeploymentItemStateManagement for every entry, but that would probably end up more expensive than constructing a stacktrace for a couple of objects - if any.